### ISSUE TITLE
Ars Technica Wonky in Every Mode

### ISSUE THREAD

**Issue by EnsignEpic** [m-khvoinitsky/dark-background-light-text-extension#159]
Last issue that I have, I swear XD
Basically, there is no good mode to use [Ars Technica](https://arstechnica.com/) with. Stylesheet Processor and Simple CSS block images, while Invert causes them to be inverted on the main page, while the ones in the main body of an article function fine.

**Comment by m-khvoinitsky** [comment]
The reason of not displaying images with Stylesheet Processor mode is
```css
background-blend-mode: multiply
```
Probably, there should be added a site-specific workaround for that which just would remove it.

**Comment by jotaf98** [comment]
Yes this is a bit annoying. Won't `background-blend-mode: multiply` be a problem with other sites? Can't the extension reset the blend mode to normal in this case?
